### PR TITLE
Add member profile

### LINKED ISSUE #5836: [Community] Member Profile of Aviral Asthana
**Current Behavior**

@Aviral0702 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @Aviral0702 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: https://github.com/Aviral0702
- Twitter: 
- LinkedIn: https://www.linkedin.com/in/aviral-asthana-02b70824b/
- Layer5 Cloud: https://meshery.layer5.io/user/3f770688-e487-45c2-943b-c94b950d3d69
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by anshikavashistha**
**Description**

This PR fixes #5836 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by anshikavashistha** [comment]
PTAL @Ashparshp

**Comment by l5io** [comment]
🚀 Preview for commit 86198ded67f13b38743a8327ccad3be39cc9988b at: https://66d4c20e824c6427ed561359--layer5.netlify.app

File: src/collections/members/aviral-asthana/Aviral-Asthana.png
Comment (Ashparshp) [inline src/collections/members/aviral-asthana/Aviral-Asthana.png:1]: Compress the image and change to .webp format.

File: src/collections/members/aviral-asthana/index.mdx
Comment (Ashparshp) [inline src/collections/members/aviral-asthana/index.mdx:1]: @Aviral0702, check the bio and everything is okay?

**Review by Ashparshp — COMMENTED** [review]
@anshikavashistha, update please.

File: src/collections/members/aviral-asthana/index.mdx
Comment (Aviral0702) [reply src/collections/members/aviral-asthana/index.mdx:1]: Please correct the LinkedIn profile else everything looks good to me.
LinkedIn : aviral-asthana-02b70824b

Comment (anshikavashistha) [reply src/collections/members/aviral-asthana/index.mdx:1]: @Aviral0702 @Ashparshp I have done the required changes.PTAL

File: src/collections/members/aviral-asthana/Aviral-Asthana.png
Comment (anshikavashistha) [reply src/collections/members/aviral-asthana/Aviral-Asthana.png:1]: @Ashparshp I have done the required changes

File: src/collections/members/aviral-asthana/index.mdx
Comment (Aviral0702) [reply src/collections/members/aviral-asthana/index.mdx:1]: @anshikavashistha Can you change the location to Ghaziabad, India.

**Comment by l5io** [comment]
🚀 Preview for commit 66141d787e188e91da36d150325c75be1350065f at: https://66d5675ab075bdb7fe6423f8--layer5.netlify.app

**Comment by anshikavashistha** [comment]
PTAL @Ashparshp

**Comment by l5io** [comment]
🚀 Preview for commit 6d7b869227f21a877cef224d81785034773b7660 at: https://66d5baf227b4c103c05080f7--layer5.netlify.app

**Review by Ashparshp — APPROVED** [review]
LGTM! Thank you @anshikavashistha.

**Review by sudhanshutech — CHANGES_REQUESTED** [review]
there is something wrong here, i couldn't see the member profile either in active state or inactive state. @anshikavashistha have you verified your change?  This is a no state
image

File: src/collections/members/aviral-asthana/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+position: Contributor
+image_path: ./Aviral-Asthana.webp
+github: Aviral0702
+linkedin: aviral-asthana-02b70824b
+layer5: 3f770688-e487-45c2-943b-c94b950d3d69
+location: Ghaziabad, India
+bio: A passionate Web Developer and Competitive Programmer from India
+status: active
+badges:
+ - First Design
```
Comment (sudhanshutech) [inline src/collections/members/aviral-asthana/index.mdx:12]: remove these badges, there no badges for profile with first design and sharing is caring. // @vishalvivekm

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: src/collections/members/aviral-asthana/index.mdx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+name: Aviral Asthana
+position: Contributor
+image_path: ./Aviral-Asthana.webp
+github: Aviral0702
+linkedin: aviral-asthana-02b70824b
+layer5: 3f770688-e487-45c2-943b-c94b950d3d69
+location: Ghaziabad, India
+bio: A passionate Web Developer and Competitive Programmer from India
+status: active
```
Comment (Ashparshp) [inline src/collections/members/aviral-asthana/index.mdx:10]: It's Active not active

**Review by Ashparshp — CHANGES_REQUESTED** [review]

File: src/collections/members/aviral-asthana/index.mdx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+---
+name: Aviral Asthana
+position: Contributor
+image_path: ./Aviral-Asthana.webp
+github: Aviral0702
+linkedin: aviral-asthana-02b70824b
+layer5: 3f770688-e487-45c2-943b-c94b950d3d69
+location: Ghaziabad, India
+bio: A passionate Web Developer and Competitive Programmer from India
```
Comment (Aviral0702) [inline src/collections/members/aviral-asthana/index.mdx:9]: Aviral Asthana is a Computer Science student with a passion for full-stack web development. Proficient in the MERN stack, Aviral has hands-on experience in building robust applications and is keenly interested in Golang, Docker, and Kubernetes. He is dedicated to continuous improvement through contributions to open-source projects and actively engaging with the developer community.

Can you please update the bio with this one?

**Review by Aviral0702 — CHANGES_REQUESTED** [review]

**Comment by sudhanshutech** [comment]
@anshikavashistha will you able to complete this?

**Comment by Ashparshp** [comment]
@anshikavashistha, please confirm if you're facing any issues with this. If you're unavailable, I will go ahead and fix it.

**Comment by anshikavashistha** [comment]
> @anshikavashistha, please confirm if you're facing any issues with this. If you're unavailable, I will go ahead and fix it.

Go ahead @Ashparshp I am not available right now

File: src/collections/members/aviral-asthana/index.mdx (hunk 3 above)
Comment (leecalcote) [inline src/collections/members/aviral-asthana/index.mdx:9]:
```suggestion
bio: Aviral Asthana is a Computer Science student with a passion for full-stack web development. Proficient in the MERN stack, Aviral has hands-on experience in building robust applications and is keenly interested in Golang, Docker, and Kubernetes. He is dedicated to continuous improvement through contributions to open-source projects and actively engaging with the developer community.
```

Comment (leecalcote) [reply src/collections/members/aviral-asthana/index.mdx:9]: Updated. 👍

File: src/collections/members/aviral-asthana/index.mdx (hunk 2 above)
Comment (leecalcote) [inline src/collections/members/aviral-asthana/index.mdx:10]:
```suggestion
status: Active
```

File: src/collections/members/aviral-asthana/index.mdx (hunk 1 above)
Comment (leecalcote) [reply src/collections/members/aviral-asthana/index.mdx:12]:
```suggestion
```

File: src/collections/members/aviral-asthana/index.mdx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+status: active
+badges:
+ - First Design
+ - Sharing is Caring
```
Comment (leecalcote) [inline src/collections/members/aviral-asthana/index.mdx:13]:
```suggestion
```

**Review by leecalcote — APPROVED** [review]

**Comment by Ashparshp** [comment]
It's completed!

**Pull request closed without merging**